src/main/java/hudson/plugins/audit_trail/AuditTrailPlugin.java

```diff
@@ -69,6 +69,8 @@ public class AuditTrailPlugin extends GlobalConfiguration {
     private boolean logBuildCause = true;
     private boolean displayUserName = false;
     private boolean logCredentialsUsage = true;
+    private boolean logScriptUsage = true;
```

Comment:
Is it worth releasing this with a default of false in case there's issues with this and flip the default later?

Reply:
Good idea!

```suggestion
    private boolean logScriptUsage = false;
```